.bsp/sbt.json

```diff
@@ -0,0 +1 @@
+{"name":"sbt","version":"1.9.7","bspVersion":"2.1.0-M1","languages":["scala"],"argv":["/Users/markgalea/Library/Java/JavaVirtualMachines/openjdk-21.0.2/Contents/Home/bin/java","-Xms100m","-Xmx100m","-classpath","/Users/markgalea/Library/Application Support/JetBrains/IntelliJIdea2024.1/plugins/Scala/launcher/sbt-launch.jar","-Dsbt.script=/usr/local/bin/sbt","xsbt.boot.Boot","-bsp"]}
```

Comment:
I suspect you don't want to check this one in -- probably belongs in .gitignore.